### PR TITLE
Implement client

### PR DESCRIPTION
**Pull request by IhorBerehovskyi**
Look at commits messages

### CODE REVIEW

File: server/src/main.cpp [hunk 1]
```diff
@@ -94,6 +91,19 @@ int main(int argc, char **argv)
...
 
+    int status = std::system(("sudo hostnamectl set-hostname " + newHostname).c_str());
+    if (status != 0) {
+        std::cerr << "Error setting hostname. Status: " << status << ". Error: " << strerror(errno)
+                  << '\n';
+        return EXIT_FAILURE;
+    }
+    status = std::system("sudo systemctl restart avahi-daemon");
+    if (status != 0) {
+        std::cerr << "Error restarting avahi-daemon. Status: " << status
```
Comment (Sage-Cat) [inline server/src/main.cpp]: Use SPDLOG (for my code also, I just forgot it)

std::cout we need only for `printUsage`

File: server/src/main.cpp [hunk 2]
```diff
@@ -94,6 +91,19 @@ int main(int argc, char **argv)
         return EXIT_FAILURE;
     }
 
+    int status = std::system(("sudo hostnamectl set-hostname " + newHostname).c_str());
```
Comment (Sage-Cat) [inline server/src/main.cpp]: Write comments about what is going on here

File: server/src/main.cpp [hunk 3]
```diff
@@ -53,27 +52,25 @@ int main(int argc, char **argv)
 {
     SpdlogConfig::init<SpdlogConfig::LogLevel::Trace>();
 
+    const std::string newHostname = "co2measure.local";
```
Comment (Sage-Cat) [inline server/src/main.cpp]: "new"?
Why "new"?

File: client/app/src/main/java/com/myproj/client/fragments/ChartFragment.kt [hunk 4]
```diff
@@ -109,44 +104,24 @@ class ChartFragment : Fragment() {
...
+                requestsCompleted++
+                if (requestsCompleted == 2) {
+                    drawChart(samplesIndoor, samplesOutdoor)
                 }
             }
 
-            override fun onFailure(call: Call<ResponseBody>, t: Throwable) {
-                // Обробка помилки
+            override fun onFailure(t: Throwable) {
+                // Handle failure
```
Comment (Sage-Cat) [inline client/app/src/main/java/com/myproj/client/fragments/ChartFragment.kt]: At least log it or make a popup for user to see (message box with error)

**Review by Sage-Cat — CHANGES_REQUESTED** [review]

File: server/src/main.cpp (hunk 3 above)
Comment (IhorBerehovskyi) [reply server/src/main.cpp]: Done

File: server/src/main.cpp (hunk 1 above)
Comment (IhorBerehovskyi) [reply server/src/main.cpp]: Done

File: client/app/src/main/java/com/myproj/client/fragments/ChartFragment.kt (hunk 4 above)
Comment (IhorBerehovskyi) [reply client/app/src/main/java/com/myproj/client/fragments/ChartFragment.kt]: Done

File: server/src/main.cpp (hunk 2 above)
Comment (IhorBerehovskyi) [reply server/src/main.cpp]: Done

**Pull request closed without merging**